lib/components/header.js

```diff
@@ -81,11 +125,60 @@ export default class Header extends Component {
       onClose: this.props.onCloseTab,
       onChange: this.onChangeIntent
     });
+    const {borderColor} = props;
+    let title = 'Hyper';
+    if (props.tabs.length === 1 && props.tabs[0].title) {
+      // if there's only one tab we use its title as the window title
+      title = props.tabs[0].title;
+    }
+    const {hambMenu, winCtrls} = this.getWindowHeaderConfig();
+    const left = winCtrls === 'left';
     return (<header
       className={css('header', isMac && 'headerRounded')}
       onClick={this.handleHeaderClick}
       onMouseDown={this.handleHeaderMouseDown}
       >
+      {
+        !isMac &&
```

Comment:
This looks a bit messy, but i guess it works. Maybe break the different cases to new stateless components?